conda-recipe/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: csxtools
+  version: {{ environ.get('GIT_DESCRIBE_TAG', 'GIT_STUB') }}.post{{ environ.get('GIT_DESCRIBE_NUMBER','') }}
```

Comment:
Tag major.minor.micro with or without the v.  It really does not matter. Inclusion/exclusion of the 'v' is totally up to you.  If you include the 'v', then we need to add a [1:] to the end of `environ.get('GIT_DESCRIBE_TAG', 'GIT_STUB')`. If you do not include the 'v' then we do not need to add a [1:] to the end.  We are tagging with a 'v'.